### DISCUSSION TITLE
Artists with 2 Deezer links

### DISCUSSION THREAD

**Discussion by dougalbob** [RandomNinjaAtk/docker-lidarr-extended#30] [General]
These are possible edge cases as I have only come across two instances so far and TBH given how infrequent I would understand entirely if you wanted to close the discussion off the bat. 

Having said that I just wonder if others may have come across similar instances were artists appear twice in Deezer, one for their primary discography and a second where they may be credited on a VA album or some such.   

K.D Lang is an example [where there are links   ](https://www.deezer.com/en/artist/5793473)

The script appears to be checking the link above and would have no reason to assume it was not the primary link to the artist.  I guess I could add album Url's for each release but if this is a more common scenerio in other genres would it be a possible candidate for a modification/additional routine in the script where two or more Deezer links are identified?

**Comment by RandomNinjaAtk** [comment]
The script is actually designed to handle multiple artist links, so it shouldn't be a problem. Now if it is incorrect, then it is causing a possible waste of time, by searching a artist library that will never match... It would be a good idea for you to remove bad urls/links from musicbrainz though when you find them.

Edit, it is not designed to handle multiple links at the album level though... So when it searches for a direct album link, it will only pull the first one.

**Reply by dougalbob** [reply]
Thanks for clarifying. It's possible then other artists I have added may have had multiple artist links and I hadn't noticed as script found what it was asked to over these multiple links.  Appears this K.D lang is some weird outlier as one of the Deezer links is the primary artist page but not picking up on any of the albums that appear in Lidarr wanted.   Did check track count in Deezer and matched this with releases Lidarr had to offer but no joy

I did previously attempt to delete the one MB link pointing to the VA credits but was ticked off by a musicbrainz mod for my trouble LOL. 

As I say this has only happened with 2 artist of the 800 or so I have so minor point and really only mentioned this in case it may have been a wider issue.

**Reply by RandomNinjaAtk** [reply]
I know working with musicbrainz can be a pain at times, but its for the good for the health of the overall community if we all pitch in to try to help maintain it because it is community driven. So I would encourage you to keep trying to help maintain it as much as possible.

**Comment by steve1977** [comment]
How / where do i add individual album links?

MBZ has something like series (e.g., with all buddha bar compilations). Is there some support for this?

**Reply by RandomNinjaAtk** [reply]
https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki/Extended-Audio-Script-Information

**Comment by steve1977** [comment]
Very clear, thanks, this is great! Beets is no longer supported, right? So, the docker setting related to beets doesn't do anything anymore?
